spec/javascripts/paste_html_to_govspeak_spec.js

```diff
@@ -0,0 +1,29 @@
+describe('PasteHtmlToGovspeak', function () {
```

Comment:
Rather than needing a JS test - since it's really only a one liner of JS, shall we go for an integration test instead and use a cucumber feature test?

I imagine to do so we'll have to fire a synthetic paste event into the page with `execute_script` or similar.